pkg/test/suite/testcase_httpserver_extended.go

```diff
@@ -20,6 +23,16 @@ func init() {
 	}
 }
 
```

Comment:
A comment explaining roughly how the parts in here are sticked together would be nice, as it is not the most self explanatory.